viewer/vue-client/src/resources/json/combinedTemplates.json

```diff
@@ -206,7 +206,7 @@
           "bibliography": [
             {
               "@type": "Library",
-              "sigel": [""]
+              "sigel": ["DIGI"]
```

Comment:
Should not be a list, right?

Reply:
You got that right! Hmm need to fix this in all templates then! Good catch